platform/android/audio_driver_opensl.cpp

```diff
@@ -189,8 +189,8 @@ void AudioDriverOpenSL::start() {
 	//cntxt.size = sizeof(pcmData);
 	/* Set arrays required[] and iidArray[] for SEEK interface
 	(PlayItf is implicit) */
-	required[0] = SL_BOOLEAN_TRUE;
-	iidArray[0] = SL_IID_BUFFERQUEUE;
+	//required[0] = SL_BOOLEAN_TRUE;
```

Comment:
That will generate a new warning though, as now `required` and `iidArray` are defined but never used. I'd just remove them if they're not used anywhere.